### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
After merging be sure to

```sh
git tag -a '1.0.0' -m '1.0.0'
git push --tags
```

File: CHANGELOG [hunk 1]
```diff
@@ -1,41 +1,44 @@
+2017-05-09 (1.0.0)
+  * Release the module as v1.0.0 (Garrett Honeycutt)
+
 2017-01-24 (0.4.1)
-	* Fix service restart issue #30 (Gjermund Jensvoll)
```
Comment (ghoneycutt) [inline CHANGELOG]: converted the hard tabs to two spaces

**Comment by ghoneycutt** [comment]
Hi @sgnl05 

Tests are passing now

**Comment by sgnl05** [comment]
Thanks @ghoneycutt !

**Pull request merged**